mass_mailing_import_mailing_list/wizard/import_mailing_list.py

```diff
@@ -0,0 +1,360 @@
+    nr_failed_rows = fields.Integer(string='Failed rows')
+    import_failed_mail_ids = fields.One2many(
+                                    comodel_name='import.failed.mail',
+                                    inverse_name='import_id')
+    is_imported = fields.Boolean()
+    import_type = fields.Selection(
+        string="Import type",
+        selection=[("adkd", "ADKd Campaign"),
+                   ("single_use", "Single-use mailing")],
+    )
+    file = fields.Binary(string='File')
+    filename = fields.Char(string='File name')
+    dwnld_csv_filename = fields.Char("CSV File name")
+    dwnld_xls_filename = fields.Char("XLS File name")
+    dwnld_txt_filename = fields.Char("TXT File name")
+    dwnld_csv_file = fields.Binary("CSV File")
+    dwnld_xls_file = fields.Binary("XLS File")
+    dwnld_txt_file = fields.Binary("TXT File")
+
+
+    @staticmethod
+    def check_header(header):
+        """
+        Verify that header has the required columns as well as create
+         a dict with indexes.
+         """
+        # Verify Header, Force it lowercase
+        header = {x.lower(): index for index, x in enumerate(header)}
+        correct_header = [
+            'activemail',
+            'sökande id'
+        ]
+        if not all(item in header for item in correct_header):
+            raise UserError(_('Please correct Header in file!\n'
+                              'Header has to contain:\n') +
+                            '\n'.join(correct_header))
+        return header
+
+    def get_contact(self, sokande_id, partner_obj, contact_obj, email):
+        """Create mass_mailing_contact and tie it to res_partner"""
+        # Need to do sudo here if res_partner.is_jobseeker is True
+        partner = partner_obj.sudo().search(
+            [('customer_id', '=', sokande_id)]
+        )
+        if not partner:
+            return
+        # Check for blacklist
+        in_mail_unsubscription = self.env['mail.unsubscription'].search(
+            [('email', '=', partner.email)]
+        )
+        if in_mail_unsubscription:
+            return
+        contact = contact_obj.search([('partner_id', '=', partner.id)])
+        if contact:
+            return contact.id
+        contact = contact_obj.create({
+            'partner_id': partner.id,
+            'name': partner.name,
+            # If partner doesn't have email then use imported email
+            'email': partner.email or email,
+        })
+        return contact.id
+
+    def get_campaign(self):
+        mailing_list = self.env['mail.mass_mailing.list'].search(
+            [('is_adkd_campaign', '=', True)]
+        )
+        if not mailing_list:
+            mailing_list = mailing_list.create(
+                {
+                    'name': _('ADKd Campaign'),
+                    'is_adkd_campaign': True
+                 })
+        elif len(mailing_list) > 1:
+            raise UserError(
+                _('There can only be one ADKd Campaign mailing list')
```

Comment:
This error message while telling what goes wrong how can the user correct it?
Can we add some more info here, can we know why it has happened?
Or can we add some more debuging information in a log message maybe?
Maybe some more info to user as well